voxaboxen/training/train_model.py

```diff
@@ -8,41 +10,70 @@
 import sys
 import os
 
+
+def print_metrics(metrics, just_one_label):
+    for pred_type in metrics.keys():
+        to_print = {k1:{k:round(100*v,4) for k,v in v1.items()} for k1,v1 in metrics[pred_type]['summary'].items()} if just_one_label else dict(pd.DataFrame(metrics[pred_type]['summary']).mean(axis=1).round(4))
+        print(f'{pred_type}:', to_print)
 def train_model(args):
+
   ## Setup
   args = parse_args(args)
 
   set_seed(args.seed)
-  
+
   experiment_dir = os.path.join(args.project_dir, args.name)
   setattr(args, 'experiment_dir', str(experiment_dir))
-  if not os.path.exists(args.experiment_dir):
+  if os.path.exists(args.experiment_dir) and args.overwrite and args.name!='demo':
+    sys.exit('experiment already exists with this name')
     os.makedirs(args.experiment_dir)
-    
+
   experiment_output_dir = os.path.join(experiment_dir, "outputs")
   setattr(args, 'experiment_output_dir', experiment_output_dir)
   if not os.path.exists(args.experiment_output_dir):
     os.makedirs(args.experiment_output_dir)
-  
+
   save_params(args)
   if hasattr(args,'stereo') and args.stereo:
     model = DetectionModelStereo(args)
   else:
     model = DetectionModel(args)
-  
+
+  if args.reload_from is not None:
```

Comment:
I think this is redundant with `args.previous_checkpoint_fp`, which currently happens in `train.py`

Reply:
oh yes I see, I added this for when n_epochs==0 (i.e. just reloading and testing some checkpoint), but yeah clumsy to have two different ways of loading checkpoints, could just make a tiny change to train.py, line 122: 

`val_epoch(model, t+1, val_dataloader, args)` ---> `val_epoch(model, args.n_epochs, val_dataloader, args)`?